### ISSUE TITLE
failed to execute querySelectorAll on document, invalid selector

### ISSUE THREAD

**Issue by larkin** [nasa/openmct#1846]
Receive a dom exception when trying to add telemetry objects to a layout, they have identifiers that include multiple characters that are not valid for a css class name.  When the layout tries to query for the class name it generates to identify new elements, it fails.

This is because the following is an invalid selector:

```
> document.querySelectorAll('.warp:packet.point-1234');
Uncaught DOMException: Failed to execute 'querySelectorAll' on 'Document': '.warp:packet.point-1234' is not a valid selector.
```

This occurred in VISTA but should be easy to reproduce with either the tutorial or the msl example telemetry.   layout.html should be updated to use mct-init-select instead of the current query-for-element workaround so that it does not rely on using object ids as class names.

**Comment by VWoeltjen** [comment]
Quickfix on #1847 using a `data-` attribute. Doesn't take the desired approach (`mct-init-select`) described above, does address the immediate issue; think this might be useful as a smallest-change to resolve the issue for testing.

**Comment by VWoeltjen** [comment]
Verified fixed during testathon 20180221, using telemetry elements with IDs like those described in the original issue

**Comment by charlesh88** [comment]
Testathon 2/21/2018: Verified fixed
